src/static/css/pages/offices.less

```diff
@@ -20,6 +20,7 @@
 
         ul {
             .list__branded();
+            padding: 0 0 0 20px;
```

Comment:
Are the top, right, and bottom padding already defined? If not, just use `padding-left`.